src/v2/Components/Artwork/Details.tsx

```diff
@@ -62,7 +62,11 @@ const ArtistLine: React.FC<DetailsProps> = ({
   }
 
   if (!artists?.length) {
-    return null
+    return (
+      <Text variant={tokens.variant} overflowEllipsis>
+        Artist Unavailable
+      </Text>
+    )
```

Comment:
There may be a situation when artists are not specified for artwork (for example, as for [this artwork](https://www.artsy.net/artwork/avalanche-full-set)) and because of this there is such a bug with hovering. We discussed this with Lois and decided that placeholder text would be rendered for such cases


#### Before
https://user-images.githubusercontent.com/3513494/173403328-50ef5cee-0c7a-44da-a84c-4eea30d60057.mp4

#### After

https://user-images.githubusercontent.com/3513494/173403401-bd30b1c7-0c82-4170-bcf8-b57809650967.mp4